src/utils/sanitizeInput.js

```diff
@@ -0,0 +1,8 @@
+export function customSanitize(input) {
+  const sanitizedInput = input
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;")
+    .replace(/'/g, "&#39;")
+    .replace(/"/g, "&quot;")
```

Comment:
Apu, for my information can you please explain why we are just replacing these symbols with this specific string like: "&lt;", "&gt;". 

If it is because, our goal with the sanitization is to replace symbol with their meaning as string? Then I would suggest to write it full. For example: Less Than instead of lt. 

If not please tell me the main reason.

Thanks.